### DISCUSSION TITLE
Will you consider updating the logo?

### DISCUSSION THREAD

**Discussion by bboydflo** [zed-industries/zed#6477] [Feature Requests]
Hi,

Thanks for this awesome project. I am taking small steps in moving from VS Code to Zed editor. That being said, In light of what's happening in the world at the moment I think the zed logo it's a bit unfortunate.

It is at least a bit annoying to see the "Z" letter when switching programs or in the dock etc. It keeps bugging me. I know it's not the most important thing, but I hope you can consider something more joyful. 

The issue with "Z" nowadays is that it's more and more associated as a symbol that supports the war in Ukraine.

**Comment by iamnbutler** [comment]
Hey! We hear you – Just wanted to make sure this didn't go unanswered. 

We have discussed this, and have brand work upcoming. We'll keep this in mind. (This isn't a promise that something specific will change, just that we hear and acknowledge what you are saying.)

> The issue with "Z" nowadays is that it's more and more associated as a symbol that supports the war in Ukraine.

Hopefully it is obvious, but this is ***not*** intentional, and is in no way affiliated with [Z](https://en.wikipedia.org/wiki/Z_(military_symbol)).

I personally created this logo and icon, so I can assure that.

Luckily, if it really bothers you for now, [macOS makes it really easy for you to change an app icon](https://9to5mac.com/2021/11/08/change-mac-icons/).

**Comment by JosephTLyons** [comment]
Our newest logo is now out in stable as of v0.76.1.
